src/pages/index.js

```diff
@@ -90,18 +92,71 @@ const DescriptiveText = styled(P)`
     color: ${colorMap.greyLightTwo};
     margin-bottom: ${size.medium};
 `;
+
+const ThumbnailCard = styled(Card)`
+    position: relative;
+`;
+
+const ThumbnailButton = styled(Button)`
+    left: 40%;
+    position: absolute;
+    top: 35%;
+`;
+
+const Thumbnail = ({ video }) => {
+    let thumbnailUrl = buildImage; //fallback image if there is none
```

Comment:
I wonder if it is possible to get a more upscaled image? This one looks a bit grainy on large devices

Reply:
Yea I'll look into a better image than this one.

Reply:
Maybe `src/images/2x/Build@2x.png`